common/source/docs/common-msp-osd-overview-4.2.rst

```diff
@@ -170,9 +170,13 @@ Features
 DisplayPort OSDs can render all the panel items supported by the ArduPilot's onboard OSD.
 Features such as multiple screen switching, multiple units and statistics are supported as well, please refer to the :ref:`onboard OSD documentation <common-osd-overview>`  for more info.
 
-By setting :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 2 to 1 one can force ArduPilot to impersonate betaflight and use a betaflight compatible font table on the remote OSD system.
-This is required if the remote OSD system does not have an ArduPilot compatible fonts table. MWOSD already supports custom fonts and therefore does not require this hack while FatShark's SharkByte will support custom fonts in a future release.
-Default behaviour is to use the ArduPilot fonts table.
+By setting :ref:`MSP_OPTIONS<MSP_OPTIONS>` bit 2 to 1 (value = 4) one can force ArduPilot to impersonate Betaflight and use a Betaflight compatible font indexes for the font table integrated in the remote OSD system.
+
+This is required if the remote OSD system does not have an ArduPilot compatible fonts table. MWOSD and DJI goggles using the wtf-os/msp-osd firmware already support custom fonts locally and therefore does not require this hack, while HDzero will support custom fonts in a future release.
```

Comment:
latest hdzero firmware does support custom ardupilot fonts, so this needs to be updated